website/docs/guide-chapter-usage.rst

```diff
@@ -11,6 +11,11 @@ frequency data (as in :numref:`data-allelecount`),
 microsatellite data, SNP data, and nucleotide and amino acid sequence
 data.
 
+As mentioned in the installation chapter, a minimal working example 
+of a :ref:`configuration file (.ini) <https://github.com/mmariani123/pypop/blob/main/tests/data/USAFEL-UchiTelle-small.pop>`, 
```

Comment:
You should change the base of the URL to be to the main repo, not your repo: https://github.com/alexlancaster/pypop/

Reply:
Also the `:ref:` syntax is for creating internal links to anchors within the document, for external URLs follow the syntax described here: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages

Reply:
Thank you for the feedback Alex, I am attempting updates now!

Reply:
Cool. If you like the changes I suggested you can just commit those as-is and can then continue to make changes (if you need any more) on top of them.

Reply:
Ah, I see you made those changes. You're almost there - see my review, below.